lib/browser/navigation/components.jsx

```diff
@@ -59,7 +59,7 @@ function Sidebar ({items, page, url_for, config, search, uncollapse, tocItems, v
             dc-icon--interactive
             doc-sidebar__vertical-menu__item
             doc-sidebar__vertical-menu__item--primary"
-        onClick={uncollapse}>
+          onClick={uncollapse}>
```

Comment:
This will break linting. Check [PR107](https://github.com/zalando-incubator/hexo-theme-doc/pull/107/files#diff-0edc4c46ab337e082f1ca4ccf67856f4R62)

Reply:
@bhaskarmelkani I don't know why but my ```lint --fix``` on my machine change that automatically..
BTW the CI job pass in any case.